openslides_votecollector/api.py

```diff
@@ -41,25 +41,19 @@ def __str__(self):
 
 def get_server():
     """
-    Gets a server proxy object and tests the connection.
+    Gets a server proxy object.
     """
     try:
-        server = ServerProxy(config['votecollector_uri'])
-        # TODO: reduce timeout
+        return ServerProxy(config['votecollector_uri'])
```

Comment:
Coding style proposal: Do not use return inside the try block. The last line of each function should be the return line:

```python
try:
    server = ServerProxy(config['votecollector_uri'])
except TypeError:
    raise VoteCollectorError(_('Server not found.'))
return server
```